user-guide/Advanced_Functionality/Swarming/Swarming.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: Swarming
+---
+
```

Comment:
I have added this as a landing page for the new Swarming section in our documentation. Could you double-check whether this is all correct, @wdm-slc?